presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/ConfigAPI.java

```diff
@@ -44,7 +42,6 @@
 public final class ConfigAPI
 {
     private static final APIFileHandler apiFileHandler = new APIFileHandler(Paths.get("/etc/presto"));
-    private static final Logger LOGGER = LogManager.getLogger(ConfigAPI.class);
```

Comment:
Why did you decide to remove logger from this class? Is the logging of the requests handled in `ApiFileHandler`? If so, I'd extract this change into a separate commit because it's not related to migration to Airlift.